Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(json: NSDictionary) {
```

Comment:
I can capitalize it but have no opinion. I usually try to keep parameters lowercased but here is json which is always capitalized as shortcut, so, okay, let's do it.